packages/cli/src/WorkflowRunner.ts

```diff
@@ -54,12 +56,48 @@ export class WorkflowRunner {
 		private readonly workflowStaticDataService: WorkflowStaticDataService,
 		private readonly nodeTypes: NodeTypes,
 		private readonly permissionChecker: PermissionChecker,
+		private readonly ownershipService: OwnershipService,
 	) {
 		if (this.executionsMode === 'queue') {
 			this.jobQueue = Container.get(Queue);
 		}
 	}
 
+	async init() {
+		if (this.executionsMode === 'queue') return;
+
+		const executionIds = await this.executionRepository.getNewExecutionIds();
+		for (const executionId of executionIds) {
+			this.logger.debug('Resuming enqueued execution', { executionId });
+			// Don't block here by using await
+			void this.reEnqueueExecution(executionId);
+		}
+	}
+
+	private async reEnqueueExecution(executionId: string) {
+		const fullExecutionData = await this.executionRepository.findSingleExecution(executionId, {
+			includeData: true,
+			unflattenData: true,
+		});
+
+		if (!fullExecutionData) {
+			throw new ApplicationError('Execution does not exist.', { extra: { executionId } });
+		}
```

Comment:
If we unify the two queries, this won't be needed.

Reply:
If I'm missing something and we do need this, we should try making the error message more informative, e.g. `Failed to find execution to resume`. Also let's not report this to Sentry?